src/core.js

```diff
@@ -159,7 +167,19 @@ let Game = {
     }
 
     return turn;
-  }
+  },
+
+  addMessage: (chat, user, message) => {
+    // Limit length of user to 20 chars and message to 200 chars
```

Comment:
Maybe consider adding ellipsis to truncate.

Alternatively, maybe use a CSS overflow style to implement this.

To manually truncate the messages using javascript seems a little hack